src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java

```diff
@@ -64,7 +64,12 @@ public void bindCommands() {
 		RobotMap.Component.driverXbox.dPad.right.whenReleased(normalDrive);
 		RobotMap.Component.driverXbox.b.onlyWhileHeld(HumanInterfaceConfig.gearAlign);
 		RobotMap.Component.driverXbox.b.whenReleased(normalDrive);
-		RobotMap.Component.teensyStick.button1.whenPressed(normalDrive);
+		RobotMap.Component.teensyStick.getButton(0).whenPressed(normalDrive);
```

Comment:
Same as what aj said for this, and the rest of the constants for what the teensy uses